src/modules/form/hooks/useFormDefinition.tsx

```diff
@@ -28,9 +28,8 @@ const useFormDefinition = (
 
   if (error) throw error;
 
-  // TODO(#6159) - turn this explicit error back on?
-  // if (!formDefinition && !loading)
-  //   throw new Error(`Form not found: ${JSON.stringify(queryVariables)} `);
+  if (!formDefinition && !loading)
```

Comment:
@gigxz do you agree with adding this back here? (followup from https://github.com/greenriver/hmis-frontend/pull/977.) The work in this PR fixes the problem in this specific case. But I'm starting to wonder whether leaving it out is a better idea in general. The bug from the PR was that this caused an explicit error to be thrown when it didn't need to be (in reality, the case notes were always attached to a form processor and could always resolve their own definition when the view modal was opened).

Reply:
Depends how the frontend handles it, sometimes they see not found and sometimes an explicit error. Could be good to send to Sentry.